addons/interact_menu/functions/fnc_initMenuReorder.sqf

```diff
@@ -0,0 +1,89 @@
+#include "script_component.hpp"
+/*
+ * Author: Dystopian
+ * Initializes "More" and "Move to Root" settings and menu.
+ *
+ * Arguments:
+ * 0: Class name <STRING>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [typeOf player] call ace_interact_menu_fnc_initMenuReorder
+ *
+ * Public: No
+ */
+
+params ["_class"];
+
+private _actionTrees = GVAR(ActSelfNamespace) getVariable _class;
+private _rootNode = [_actionTrees, ["ACE_SelfActions"]] call FUNC(findActionNode);
+private _rootActions = _rootNode select 1;
+private _settingCategoryPrefix = format ["ACE %1 - ", LELSTRING(Interaction,InteractionMenuSelf)];
+
+
+// init "Move to Root" settings
+private _settingCategory = _settingCategoryPrefix + localize "STR_3DEN_Display3DEN_RemoveLayer_tooltip";
+
+private _fnc_processNode = {
+    params ["_node", "_parentVarName", "_titlePrefix", "_parentConditionString"];
+
+    _node params ["_actionData", "_actionChildren"];
+    _actionData params ["_name", "_title", "", "", "_condition"];
+    private _varName = _parentVarName + "__" + _name;
+    private _titleFull = _titlePrefix + _title;
+    private _conditionFullString = format ["%1 && {%2}", _parentConditionString, _condition call EFUNC(common,codeToString)];
+
+    // don't add already added setting
+    if (isNil {missionNamespace getVariable _varName}) then {
+        [_varName, "CHECKBOX", _titleFull, [_settingCategory, _rootActionTitle], false, false, {}, true] call CBA_fnc_addSetting;
```

Comment:
```suggestion
        [_varName, "CHECKBOX", _titleFull, [_settingCategory, _rootActionTitle], false, 2, {}, true] call CBA_fnc_addSetting;
```
Should we do this? So people don't force those settings when they export them all?

Reply:
I think this makes sense since these settings are purely individual preference.